ci/src/main/java/group10/WebsiteController.java

```diff
@@ -0,0 +1,34 @@
+package group10;
+
+import spark.Request;
+import spark.Response;
+
+import spark.ModelAndView;
+import spark.template.velocity.VelocityTemplateEngine;
+import java.util.HashMap;
+import java.util.Map;
+
+public class WebsiteController {
+
+    /**
+     * Handle the Website get event: get info about build history from database
+     * return response.
+     * @param request from website to server
+     * @param response send back info from database
+     * @return
+     */
+    public static String handleGet(Request request, Response response) 
+    {
+        Map<String, Object> model = new HashMap<>();
+        //todo: get data from DB
+        
+        return new VelocityTemplateEngine().render(new ModelAndView(model, "public/template.html"));
```

Comment:
Replace "public/template.html" with "remakeToTemplate.html" and delete template.html.